### PR TITLE
Add doc with schedule for Alma publishing jobs

### LINKED ISSUE #1759: Include in the docs the schedule of the publishing jobs in alma

### PR DESCRIPTION
**Pull request by christinach**
closes #1759

### CODE REVIEW

File: docs/alma_publishing_jobs_schedule.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Schedule of Publishing Jobs in Alma
+
+## Incremental Job
+* The job initializes daily every 6 hours, at 4:00am, 10:00am, 4:00pm, 10:00pm. It takes around 60 min to complete and get published.
```
Comment (maxkadel) [inline docs/alma_publishing_jobs_schedule.md:4]: Time zone? Does it change in EDT vs. EST?

**Review by maxkadel — CHANGES_REQUESTED** [review]
Thanks for adding this - since we've had a number of time zone issues / questions, can you add the time zone?

File: docs/alma_publishing_jobs_schedule.md (hunk 1 above)
Comment (christinach) [reply docs/alma_publishing_jobs_schedule.md:4]: I'll update it is Princeton time

Comment (maxkadel) [reply docs/alma_publishing_jobs_schedule.md:4]: Sounds good, thanks!

**Review by maxkadel — APPROVED** [review]

**Pull request merged**